src/convertToObject.js

```diff
@@ -7,6 +7,19 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString
+    .split(';')
+    .reduce((acc, declaration) => {
```

Comment:
[CODE KNOWLEDGE] - There is no need to make a copy before each operation. Methods like `split` and `reduce` do not mutate the original array. You can chain them directly.